es-ds-components/components/es-zip-code-form.vue

```diff
@@ -85,13 +85,13 @@ const rules = {
 
 const { v$, validateState } = useEsForms(rules, state);
 
-const ctaForm = ref<HTMLFormElement>('ctaForm');
+const ctaForm = useTemplateRef('ctaForm');
```

Comment:
that previous `HTMLFormElement` type comes in handy with upgrading

Reply:
I'm actually not even sure how or if it was working before, given that [template refs](https://vuejs.org/guide/essentials/template-refs) are invoked using `useTemplateRef`, and [ref](https://vuejs.org/guide/essentials/reactivity-fundamentals#declaring-reactive-state-1) is something else entirely.

I've added back the `<HTMLFormElement>` though.